ddl/ddl_worker.go

```diff
@@ -324,7 +324,6 @@ func (d *ddl) addBatchDDLJobs2Queue(tasks []*limitJobTask) error {
 
 		for i, task := range tasks {
 			job := task.job
-			job.Version = currentVersion
```

Comment:
This field was originally 1, now it's 0 after you delete this, and then if someone uses it, they might not know that they start with 1. Whether to think about this.